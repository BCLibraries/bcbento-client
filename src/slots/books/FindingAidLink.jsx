import React from 'react';
import FindingAidIcon from './icon_archives.svg';

function FindingAidLink({url}) {
    return <div className="finding-aid-link">
        <img src={FindingAidIcon} className="finding-aid-link__icon" alt=""/>
        <a href={url} target="_blank" rel="noopener noreferrer">
            Finding aid
            <i className="fa fa-external-link" aria-hidden="true"/>
        </a>
    </div>

}

export default FindingAidLink;