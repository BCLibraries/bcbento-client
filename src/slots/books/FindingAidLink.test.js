import renderer from 'react-test-renderer';
import FindingAidLink from "./FindingAidLink";

it('renders correctly', () => {
    const component = renderer.create(
        <FindingAidLink url={'https://library.bc.edu'} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
