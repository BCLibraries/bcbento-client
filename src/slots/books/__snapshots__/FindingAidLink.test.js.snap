// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="finding-aid-link"
>
  <img
    alt=""
    className="finding-aid-link__icon"
    src="icon_archives.svg"
  />
  <a
    href="https://library.bc.edu"
    rel="noopener noreferrer"
    target="_blank"
  >
    Finding aid
    <i
      aria-hidden="true"
      className="fa fa-external-link"
    />
  </a>
</div>
`;
